### ISSUE TITLE
Doesn't work with rvm, 'scssLintExecutablePath' is set

### ISSUE THREAD

**Issue by jblandry** [AtomLinter/linter-scss-lint#13]
Hi,

I'm using rvm for Ruby and even when I set the 'scssLintExecutablePath' it doesn't seem to work

Here is the output from the linter debug

```js

command and arguments [
  0: "/Users/jblandry/.rvm/gems/ruby-2.1.3/bin/scss-lint"
  1: "--format=XML"
  2: "-c"
  3: "/Users/jblandry/Desktop/www/project/.scss-lint.yml"
  4: "/var/folders/4f/8l984hdj5sxdz_93jn0vwfp0js2d50/T/AtomLinter114104-2385954pcn4source.css.scss/common.scss"
]


is node executable: false

stderr env: ruby_executable_hooks: No such file or directory

linter returned
LinterScssLint {
  cmd: "scss-lint --format=XML -c /Users/jblandry/Desktop/www/project/.scss-lint.yml"
  cwd: "/Users/jblandry/Desktop/www/project/styles"
  editor: TextEditor
  executablePath: "/Users/jblandry/.rvm/gems/ruby-2.1.3/bin"
}

command `/Users/jblandry/.rvm/gems/ruby-2.1.3/bin/scss-lint` timed out after 5s 

```

Thanks

**Comment by cvknage** [comment]
I am experiencing the same issue. 

Here it the output from the Atom Console:

```
activate linter-scss-lint

Window load time: 1133ms

command and arguments [
   0: "/Users/chris/.rvm/gems/ruby-2.1.2/bin/scss-lint"
   1: "--format=XML"
   2: "/var/folders/l2/nrlwyt_95tb786rrx02_y8f40000gs/T/AtomLinter115123-18502-1ijzsd9source.css.scss/_theme.scss"
   length: 3
   __proto__: Array[0]
]

is node executable: false

stderr env: ruby_executable_hooks: No such file or directory

scss-lint returned 
LinterScssLint {
   cmd: "scss-lint --format=XML"
   cwd: "/Users/chris/Desktop/cvk-angular-seed/app/styles"
   editor: TextEditor
   executablePath: "/Users/chris/.rvm/gems/ruby-2.1.2/bin/scss-lint"
   executionTimeout: 5000
   subscriptions: CompositeDisposable
   __proto__: ctor
}
```

**Comment by hejhi** [comment]
yup, confirmed. Same as the above. Would love it to work with my rvm :'(

**Comment by caillou** [comment]
Same problem here. Did anyone find a solution to this?

**Comment by steelbrain** [comment]
Fixed by https://github.com/AtomLinter/linter-scss-lint/pull/58

**Comment by Jpunt** [comment]
Are the fixes by #58 released already? I've got the same error with current version of this plugin.
